features/providers/complete_and_check_means_answers.feature

```diff
@@ -5,54 +5,100 @@ Feature: Completing and checking means answers backwards and forwards
     Given The means questions have been answered by the applicant
```

Comment:
means question are no longer answered byt eh applicant, right? in which case this step needs to be replaced I am guessing?!